lib/builder/builder.js

```diff
@@ -249,7 +250,12 @@ module.exports = {
 			virBasePath: "/"
 		});
 
-		const buildContext = new BuildContext({rootProject: tree});
+		const buildContext = new BuildContext({
+			rootProject: tree,
+			options: {
+				cssVariables: cssVariables
```

Comment:
I wonder whether we should have some pre-fixes for the options key (similar to the tags), in case we want to allow a more generic approach for custom tasks.
But as we don't document the "cssVariables" option as API, we should be able to also do this later on, so IMO this is fine.

Reply:
So you mean create a `BuildOptionsCollection` like `ResourceTagCollection`?